server.ts

```diff
@@ -1,48 +1,61 @@
-import 'zone.js/dist/zone-node';
 
-import { ngExpressEngine } from '@nguniversal/express-engine';
-import * as express from 'express';
-import { join } from 'path';
+import 'zone.js/node';
 
-import { AppServerModule } from './src/main.server';
 import { APP_BASE_HREF } from '@angular/common';
-import { existsSync } from 'fs';
+import { CommonEngine } from '@angular/ssr';
+import * as express from 'express';
+import { existsSync } from 'node:fs';
+import { join } from 'node:path';
+import bootstrap from './src/main.server';
 
+// The Express app is exported so that it can be used by serverless Functions.
```

Comment:
The style/format for inline comments is that they start lowercase and have no punctuation at the end:
```suggestion
// the Express app is exported so that it can be used by serverless functions
```